installing/installing_ibm_cloud_public/installing-ibm-cloud-restricted.adoc

```diff
@@ -0,0 +1,84 @@
+:_content-type: ASSEMBLY
+[id="installing-ibm-cloud-restricted"]
+= Installing a cluster on IBM Cloud in a restricted network
+include::_attributes/common-attributes.adoc[]
+:context: installing-ibm-cloud-restricted
+
+toc::[]
+
+In {product-title} {product-version}, you can install a cluster in a restricted network by creating an internal mirror of the installation release content that is accessible to an existing Virtual Private Cloud (VPC) on {ibm-cloud-name}.
+
+[id="prerequisites_installing-ibm-cloud-restricted"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You xref:../../installing/installing_ibm_cloud_public/installing-ibm-cloud-account.adoc#installing-ibm-cloud-account[configured an IBM Cloud account] to host the cluster.
+* You have a container image registry that is accessible to the internet and your restricted network. The container image registry should mirror the contents of the {product-registry} and contain the installation media. For more information, see xref:../../installing/disconnected_install/installing-mirroring-disconnected.adoc#installing-mirroring-disconnected[Mirroring images for a disconnected installation using the oc-mirror plugin].
+* You have an existing VPC on {ibm-cloud-name} that meets the following requirements:
+** The VPC contains the mirror registry or has firewall rules or a peering connection to access the mirror registry that is hosted elsewhere.
+** The VPC can access {ibm-cloud-name} service endpoints using a public endpoint. If network restrictions limit access to public service endpoints, evaluate those services for alternate endpoints that might be available. For more information see xref:../../installing/installing_ibm_cloud_public/installing-ibm-cloud-restricted.adoc#access-to-ibm-service-endpoints_installing-ibm-cloud-restricted[Access to IBM service endpoints].
+
++
+You cannot use the VPC that the installation program provisions by default.
+* If you plan on configuring endpoint gateways to use {ibm-cloud-name} Virtual Private Endpoints, consider the following requirements:
+** Endpoint gateway support is currently limited to the `us-east`, `us-south`, and `eu-de` regions.
```

Comment:
@mjpytlak @jeffnowicki can you confirm `eu-de` region here? When I am installing a disconnected cluster in `eu-de` region, then overriding `ResourceController` serviceEndpoints to `https://private.resource-controller.cloud.ibm.com/`, but I did not find a endpoint gateway targeted to ``private.resource-controller.cloud.ibm.com` in the output of `ibmcloud is endpoint-gateway-targets --output JSON` against `eu-de` region.

Reply:
> can you confirm `eu-de` region here? When I am installing a disconnected cluster in `eu-de` region, then overriding `ResourceController` serviceEndpoints to `https://private.resource-controller.cloud.ibm.com/`, but I did not find a endpoint gateway targeted to ``private.resource-controller.cloud.ibm.com` in the output of`ibmcloud is endpoint-gateway-targets --output JSON` against `eu-de` region.

@jianlinliu you are correct, I confirmed that is the one service in `eu-de` that's missing a private endpoint and would be needed for a "fully disconnected" cluster installation.

@mjpytlak - please remove mention of `eu-de` from that section of the docs.

fyi @cjschaef

Reply:
> please remove mention of `eu-de` from that section of the docs.

Removed. Thank you.